src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -10273,6 +10276,12 @@ namespace AirflowNetwork {
                     NodeFound(i) = true;
                     StandaloneERVFound = true;
                 }
+
+                // Skip Window AC with no OA
+                if (GetWindowACNodeNumber(m_state, i)) {
+                    NodeFound(i) = true;
+                    WindowACFound = true;
+                }
```

Comment:
Use the same approach as before by calling a function that checks if a node is part of a `ZoneHVAC:WindowAirConditioner` and if so allows it to exist without being part of the AFN.